concourse/pipeline/job_def.lib.yml

```diff
@@ -128,5 +148,47 @@ plan:
     resource: #@ conf["res_gpdb_src"]
   - get: bin_plpython3
     resource: #@ conf["res_plpython3_bin"]
-- #@ _test_task(conf)
+- #@ _test_and_build_task(conf)
+- put: #@ conf["res_intermediates_bin"]
+  params:
+    file: greenplumpython_artifacts/greenplumpython.tar.gz
 #@ end
+
+#! The final release job
+#! 1. Push the artifacts to the release bucket
+#! 2. Push the git tag
+#@ def exit_release_job(param):
+#@   trigger = param["trigger"]
+#@   confs = param["confs"]
+#@   passed_jobs = []
+#@   res_map = param["res_map"]
+#@   for conf in confs:
+#@     passed_jobs.append(build_test_job_name(conf))
+#@     add_res_by_name(res_map, conf["release_bin"])
+#@   end
+name: exit_release
+on_failure: #@ trigger["on_failure"]
+on_error: #@ trigger["on_error"]
+on_success: #@ trigger["on_success"]
+plan:
+#@   for to_get in trigger["to_get"]:
+- passed: #@ passed_jobs
+  _: #@ template.replace(to_get)
+#@   end
+- in_parallel:
+    steps:
+#@   for i, conf in enumerate(confs):
+      - do:
+        - get: #@ conf["res_intermediates_bin"]
+          passed:
+            - #@ passed_jobs[i]
+          params:
+            unpack: true
+        - put: #@ conf["release_bin"]
+          params:
+            file: #@ conf["res_intermediates_bin"] + "/greenplumpython_*_*.tar.gz"
```

Comment:
this won't work since the unpacked file will be a `.whl` file
You could change the release bucket and test it.